packages/mco/components/disaster-recovery/create-dr-policy/create-dr-policy.tsx

```diff
@@ -267,22 +273,28 @@ export const CreateDRPolicy: React.FC<ReRouteResourceProps> = ({
           )}
           isHelperTextBeforeField
         >
-          <SelectClusterList state={state} dispatch={dispatch} />
-        </FormGroup>
-        <FormGroup fieldId="policy-name">
-          <Alert
-            className="co-alert mco-create-data-policy__alert"
-            title={t(
-              'OpenShift Data Foundation {{ version }} or above must be installed on the managed clusters to setup connection for enabling replication/mirroring.',
-              { version: ODF_MINIMUM_SUPPORT }
-            )}
-            variant={AlertVariant.info}
-            isInline
-          ></Alert>
+          <SelectClusterList
+            state={state}
+            requiredODFVersion={odfMCOVersion}
+            dispatch={dispatch}
+          />
         </FormGroup>
-        {!!clustersData.length && (
+        {odfMCOVersion && (
```

Comment:
it's good practice to have this:
```suggestion
        {!!odfMCOVersion && (
```